### PR TITLE
Remove database abstraction.

### PR DESCRIPTION
**Pull request by V1122am**
Signed-off-by: Vamshi <vamc5458@gmail.com>

### CODE REVIEW

**Comment by jbobo** [comment]
you might want to confirm that ledger-sync is working locally. Travis is failing tests in `tests/integration/providers/ldap/delta_inbound_sync_tests.py`, which could indicate that ledger-sync is not inserting entries from the sawtooth changefeed into their respective rethinkdb tables.

File: rbac/ledger_sync/inbound/listener.py [hunk 1]
```diff
@@ -108,31 +104,30 @@ def listener():
...
+                process(rec["new_val"], conn)
 
     except Exception as err:  # pylint: disable=broad-except
         LOGGER.exception("Inbound listener %s exception", type(err).__name__)
         LOGGER.exception(err)
 
     finally:
         try:
-            database.disconnect()
+            conn.disconnect()
```
Comment (nadiabahrami) [inline rbac/ledger_sync/inbound/listener.py]: There is no disconnect() function for rethinkdb that I am aware of.  There is a close() function, can you verify that this exists? or change it to close()?

File: rbac/ledger_sync/outbound/listener.py [hunk 2]
```diff
@@ -44,10 +47,39 @@ def listener():
 
     finally:
         try:
-            database.disconnect()
+            conn.disconnect()
```
Comment (nadiabahrami) [inline rbac/ledger_sync/outbound/listener.py]: Same here as above

Comment (V1122am) [reply rbac/ledger_sync/outbound/listener.py]: yes, there is a close function, i have changed it.

File: rbac/ledger_sync/deltas/handlers.py [hunk 3]
```diff
@@ -48,21 +47,21 @@ def update_database(database, state_change):
...
     """
     try:
         # Check for and resolve forks
         state_change.block_num = int(state_change.block_num)
-        old_block = database.fetch("blocks", state_change.block_num)
+        old_block = r.table("blocks").get(state_change.block_num).run(conn)
         if old_block is not None:
             if old_block["block_id"] != state_change.block_id:
-                drop_results = database.drop_fork(state_change.block_num)
+                drop_results = conn.drop_fork(state_change.block_num)
```
Comment (mtn217) [inline rbac/ledger_sync/deltas/handlers.py]: The connection doesn't have a drop_fork() function. You would need to call the drop_fork() function that you created in line 98 of this file.

**Pull request merged**